packages/core/src/components/Image/index.tsx

```diff
@@ -0,0 +1,224 @@
+import React from 'react';
+import { mutuallyExclusiveTrueProps } from 'airbnb-prop-types';
+import withStyles, { WithStylesProps } from '../../composers/withStyles';
+
+const ANIMATION_DURATION_MS = 300;
+const IMAGE_BORDER_RADIUS = 3;
+
+const bottomBorderPropType = mutuallyExclusiveTrueProps('hasBorderBottomRadius', 'hasBorderRadius');
+const topBorderPropType = mutuallyExclusiveTrueProps('hasBorderTopRadius', 'hasBorderRadius');
+const fitPropType = mutuallyExclusiveTrueProps('fit', 'background');
+
+export type Props = {
+  /** Alternate text if image cannot be displayed */
+  alt: string;
+  /** Specified when image is not directly a part of the content */
+  background?: boolean;
+  /** Sets top or bottom background position */
+  backgroundPositionY?: string;
+  /** Contain vs Cover for background image */
+  backgroundSize?: string;
+  /** For cross-origin resources */
+  crossOrigin?: boolean;
+  /** Align item inside container when `background` is not specified */
+  fit?: boolean;
+  /** Has rounded bottom corners */
+  hasBorderBottomRadius?: boolean;
+  /** All corners rounded. If specified, using `hasBorderBottomRadius` or `hasBorderTopRadius` is redundant */
+  hasBorderRadius?: boolean;
```

Comment:
Use case for having top and bottom separate things?

Reply:
curious too ^ to be more consistent with lunar, these props wouldn't be here. instead, just a `borderRadius` prop that defaults to `theme.ui.borderRadius` number or whatever number is passed in (see ResponsiveImage as an example http://airbnb.io/lunar/?path=/story/core-responsiveimage--an-image-that-is-constrained-proportionally)

Reply:
Because I was moving this over from DLS, I wanted to keep any functionality that could be used. But after looking through our use cases (always use `hasBorderRadius`) I just switched to the paradigm Stefanie mentioned. If anyone ever needs the top and bottom split out it can be added in later👍